app/controllers/foods_controller.rb

```diff
@@ -42,31 +41,29 @@ def update
 
   def destroy
     food = Food.find(params[:id])
-    if @current_user.role === "admin" && food
+    if food
       food.destroy
       flash[:success] = "#{food.name} has been deleted."
-      redirect_to admin_foods_path
+      redirect_to dashboard_path
     else
       flash[:error] = "An error occured. Try deleting #{@food.name} again."
     end
   end
 
   def edit_status
-   if @current_user.role === "admin"
-     food = Food.find(food_params[:id].to_i)
-     if (!food.nil?)
-       status = food_params[:status] == "true" ? "available" : "not available"
-       food.status = status
-       food.save
-       render json: food
-     end
+   food = Food.find(food_params[:id].to_i)
+   if (!food.nil?)
+     status = food_params[:status] == "true" ? "available" : "not available"
+     food.status = status
+     food.save
+     render json: food
    end
   end
 
   private
 
   def food_params
-    params.require(:food).permit(:id, :name, :description, :price, :category_id, :food_image, :status)
+    params.require(:food).permit(:id, :name, :description, :price, :category_id, :food_image, :status, {sales: [:percentage, :price, :status, :js]})
```

Comment:
Line is too long. [148/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.